TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/Box.kt

```diff
@@ -0,0 +1,48 @@
+package org.firstinspires.ftc.teamcode.api
+
+import com.qualcomm.robotcore.eventloop.opmode.OpMode
+import com.qualcomm.robotcore.hardware.Servo
+
+object Box : API() {
+    private lateinit var boxL: Servo
+    private lateinit var boxR: Servo
+    private lateinit var grip: Servo
+
+    override fun init(opMode: OpMode) {
+        super.init(opMode)
+
+        this.boxL = this.hardwareMap.get(Servo::class.java, "boxL")
+        this.boxR = this.hardwareMap.get(Servo::class.java, "boxR")
```

Comment:
Just a note that this may need to be reversed, depending on how the box is built.

Reply:
noted.